### PR TITLE
Test app:getpath occ command

### LINKED ISSUE #33614: Test app:getpath
Test that `app:getpath` returns correct path for the app.
Part of the #33054

### PR DESCRIPTION
**Pull request by skshetry**
## Description

Tests that `app:getpath` returns path for the app from `apps_paths` in the config.

## Related Issue

- Fixes #33614

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 1]
```diff
@@ -585,6 +598,29 @@ public function theAppNameReturnedByTheOccCommandShouldBe($appName) {
 		PHPUnit_Framework_Assert::assertEquals($appName, \key($lastOutputArray['apps']));
 	}
 
+	/**
+	 * @Then the path returned by the occ command should be inside one of the apps path in the config
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/OccContext.php]: s/inside one of the apps path in the config/inside one of the apps paths in the config/

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 2]
```diff
@@ -585,6 +598,29 @@ public function theAppNameReturnedByTheOccCommandShouldBe($appName) {
...
+		$configOutputArray = \json_decode($lastOutput, true);
+
+		$appPaths = $configOutputArray['system']['apps_paths'];
+
+		foreach ($appPaths as $path) {
+			if (\substr_compare($appPath, $path['path'], 0)) {
+				return;
+			}
+		}
+		throw new Exception(__METHOD__ . "App's was not found in the config.");
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/OccContext.php]: Maybe say:
"App path $appPath was not found in the config."
That will help someone a bit if the test fails.

**Comment by skshetry** [comment]
@phil-davis fixed, please review again

**Review by phil-davis — DISMISSED** [review]

**Comment by phil-davis** [comment]
I be broke:
```
  Scenario: admin checks the path of the given app                                                # /drone/src/tests/acceptance/features/cliProvisioning/getApps.feature:37
    When the administrator checks the location of the "testing" app using the occ command         # OccContext::theAdministratorChecksTheLocationOfTheAppUsingTheOccCommand()
    Then the command should have been successful                                                  # FeatureContext::theCommandShouldHaveBeenSuccessful()
    And the path returned by the occ command should be inside one of the apps paths in the config # OccContext::thePathReturnedByTheOccCommandShouldBeInsideOneOfTheAppsPathInTheConfig()
      Notice: Undefined index: apps_paths in /drone/src/tests/acceptance/features/bootstrap/OccContext.php line 614
```

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@skshetry please make backport

**Pull request merged**